pennylane/math/single_dispatch.py

```diff
@@ -87,6 +87,14 @@ def _cond(pred, true_fn, false_fn, args):
 ar.register_function("numpy", "cond", _cond)
 ar.register_function("builtins", "cond", _cond)
 
+ar.register_function("numpy", "gamma", lambda x: _i("scipy").special.gamma(x))
+ar.register_function("numpy", "gammainc", lambda x, y: _i("scipy").special.gammainc(y, x))
+
+ar.register_function("builtins", "gamma", lambda x: _i("autograd.scipy").special.gamma(x))

```

Comment:
This one can probably use 'scipy' instead of 'autograd.scipy' because there is only one argument.